src/containers/teamPage/index.tsx

```diff
@@ -0,0 +1,268 @@
+    team_role: TeamRole.MEMBER,
+  },
+  {
+    user_id: 7,
+    username: 'apradhan12',
+    team_role: TeamRole.MEMBER,
+  },
+  {
+    user_id: 8,
+    username: 'SofieDunt',
+    team_role: TeamRole.MEMBER,
+  },
+];
+
+const sampleGoalData: GoalProps[] = [
+  {
+    id: 1,
+    goal: 50,
+    progress: 5,
+    start_date: new Date(2021, 0, 1),
+    complete_by: new Date(2022, 0, 1),
+    completion_date: null,
+  },
+  {
+    id: 2,
+    goal: 500,
+    progress: 71,
+    start_date: new Date(2021, 2, 20),
+    complete_by: new Date(2021, 4, 21),
+    completion_date: null,
+  },
+  {
+    id: 3,
+    goal: 10,
+    progress: 10,
+    start_date: new Date(2020, 5, 10),
+    complete_by: new Date(2020, 7, 10),
+    completion_date: new Date(2020, 6, 13),
+  },
+];
+
+const sampleTeamData: TeamProps = {
+  id: 1,
+  name: 'Code4Community',
+  bio: 'Short description about how cool of a team Code4Community is!',
+  members: sampleMemberData,
+  goals: sampleGoalData,
+};
+
+const TeamPage: React.FC = () => {
+  const onClick = () => {
+    // TODO: join team
+  };
+
+  return (
+    <TeamContainer>
+      <StyledLinkButton to={Routes.LOGIN}>
+        {`<`} Return to Teams
+      </StyledLinkButton>
+      <TeamHeaderContainer>
+        <PageHeaderContainer>
+          <PageHeader
+            pageTitle={sampleTeamData.name}
+            pageSubtitle={sampleTeamData.bio}
+          />
+        </PageHeaderContainer>
+        <JoinButton type="primary" onClick={onClick}>
+          Join Team
+        </JoinButton>
+      </TeamHeaderContainer>
+
+      <CenterDiv>
+        <MemberContainer>
+          <SectionHeader>TEAM MEMBERS</SectionHeader>
+          <StyledMemberList
+            dataSource={sampleTeamData.members}
+            itemLayout="vertical"
+          >
+            {sampleTeamData.members.map((member) => {
+              if (
+                member.team_role === TeamRole.LEADER ||
+                member.team_role === TeamRole.MEMBER
+              ) {
+                return (
+                  <TeamMember
+                    id={member.user_id}
+                    team_role={member.team_role}
+                    username={member.username}
+                  />
+                );
+              }
+              return <></>;
+            })}
+          </StyledMemberList>
+        </MemberContainer>
+        <GoalContainer>
+          <SectionHeader>TEAM GOALS</SectionHeader>
+          <LeaderboardCollapse
+            bordered={false}
+            key={1}
+            defaultActiveKey={sampleTeamData.goals[0].id}
+            expandIconPosition={'right'}
+          >
+            {sampleTeamData.goals.map((item) => {
+              return (
+                <StyledPanel
+                  key={item.id}
+                  header={<BlackText>Goal {item.id}</BlackText>}
+                  extra={
+                    item.completion_date == null && (
+                      <BlackText>Incomplete</BlackText>
```

Comment:
❓ What if the goal is complete?

Reply:
The goal is still listed but doesn't say 'Incomplete' next to the dropdown (in the screenshots, the bottom goal is complete)